names/names.go

```diff
@@ -110,7 +110,8 @@ var (
 		{"Ebs", "EBS", "ebs", nil},
 		{"Ec2", "EC2", "ec2", nil},
 		// Prevent "Secret" from becoming "s_ecr_et"
-		{"Ecr", "ECR", "ecr", re2.MustCompile("(?!S|s)ecr(?!et)", re2.None)},
+		// Prevent "Decrease" from becoming "d_ecr_ease"
+		{"Ecr", "ECR", "ecr", re2.MustCompile("(?!S|s|D|d)ecr(?!et|ease)", re2.None)},
```

Comment:
The test case `{"Ecr", "ECR", "ecr", "ecr", "ecr"},` does not pass with this regex

Reply:
apply new regex, but just do try and error 😓 ,   if you have idea to improve the regex,  please let me know.

Reply:
Shoot.. i just saw this. Glad you figured it out!